src/workerd/api/BUILD.bazel

```diff
@@ -177,6 +213,7 @@ kj_test(
     src = "data-url-test.c++",
     deps = [
         ":data-url",
+        ":encoding",
```

Comment:
The test doesn't depend on encoding either. The same goes for api-rtti-test – encoding.h is included in there but not actually needed I think.